Engine/Results/LiveTradingResultHandler.cs

```diff
@@ -1019,6 +1021,12 @@ public void ProcessSynchronousEvents(bool forceProcess = false)
                             //Sample Portfolio Value:
                             var price = subscription.RealtimePrice;
 
+                            var now = DateTime.UtcNow.ConvertFromUtc(security.Exchange.TimeZone);
+                            if (!security.Exchange.Hours.IsOpen(now, security.IsExtendedMarketHours)) continue;
```

Comment:
Does security.Exchange.ExchangeOpen not work since no internal local frontier updates?